src/dashboard/app.py

```diff
@@ -4,6 +4,16 @@
 from datetime import datetime
 from io import BytesIO
 from typing import List
+from flask import Flask, render_template
+
+app = Flask(__name__)
+
+@app.route('/')
+def index():
+    return render_template('navigation.html')
+
+if __name__ == '__main__':
+    app.run(debug=True)
```

Comment:
are you kidding me, you added flask HTML render in streamlit app, have even checked your code